src/elements/ComicScroll.tsx

```diff
@@ -1,19 +1,19 @@
 import ComicCard from "../atoms/ComicCard";
-import { IComicData } from "../models/comicData";
+import { ICarouselComic } from "../models/homePageData";
 function ComicScroll({
   heading,
   comics,
 }: {
   heading: string;
-  comics: IComicData[];
+  comics: ICarouselComic[];
 }) {
   return (
     <>
       <h4 className="text-medium font-medium">{heading}</h4>
       <div className="major-cities-scroll bg-white overflow-auto whitespace-nowrap mb-6">
         {comics.map((comic) => (
           <a
-            key={comic.name}
+            key={comic.id}
             className="inline-block text-black text-center no-underline p-3.5"
             href="#"
           >
```

Comment:
Safety/Error checks here as well possibly?

Reply:
Okay noted, I am working on it in another PR.